### DISCUSSION TITLE
Welcome to Open Identity Platform Community Discussions!

### DISCUSSION THREAD

**Discussion by vharseko** [OpenIdentityPlatform/.github#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aaronnim** [comment]
HI, I'm researching about the OpenIdentity platform and saw that the website is down, why is that? Where can I get more info about your project?

**Reply by maximthomas** [reply]
Hi, @aaronnim, Please check again https://www.openidentityplatform.org. It's working fine

**Reply by aaronnim** [reply]
Hi, @maximthomas, I've tried on my phone in chrome and in my laptop. I also used brave browser but still no luck. The website is still down

**Reply by maximthomas** [reply]
Hi, @aaronnim I can't reproduce the issue:
```
curl -I -X GET https://www.openidentityplatform.org
HTTP/2 200 
server: GitHub.com
content-type: text/html; charset=utf-8
last-modified: Thu, 11 Jul 2024 10:56:43 GMT
access-control-allow-origin: *
etag: "668fba6b-5df0"
expires: Sat, 13 Jul 2024 07:00:35 GMT
cache-control: max-age=600
x-hosts-log-append: pages_hosts_ips:
x-proxy-cache: MISS
x-github-request-id: 5CE4:15610C:47F183:4A518C:669223BA
accept-ranges: bytes
date: Sat, 13 Jul 2024 12:59:46 GMT
via: 1.1 varnish
age: 439
x-served-by: cache-bma1682-BMA
x-cache: HIT
x-cache-hits: 2
x-timer: S1720875587.527588,VS0,VE1
vary: Accept-Encoding
x-fastly-request-id: d008d5f9063b339b806990267ba2a9ff032269f8
content-length: 24048
```

**Reply by aaronnim** [reply]
@maximthomas do you guys have communities where I can talk to others like in discord or telegram? I can't access the website even with cellular data.

**Reply by vharseko** [reply]
[Website Availability Test - Check Website Uptime | Uptimia](https://www.uptimia.com/website-availability-test/cbb96016)

**Reply by aaronnim** [reply]
It's solved now. I think I had an issue with cookies. 

One question, is the open identity platform a tool or just a collection of open source products?

**Reply by vharseko** [reply]
collection of open source products 
[![Open Identity Platform](https://www.openidentityplatform.org/assets/img/oip-star.png?ver3)](https://github.com/OpenIdentityPlatform)

**Comment by ZackSilva** [comment]
I do not see evidence of a third-party audit having been conducted against OpenAM Community Edition. Does it have any past audits or standing accreditations in any capacity?
